build.gradle

```diff
@@ -6,17 +6,39 @@ apply plugin: 'com.gradle.build-scan'
 
 // Adds support for Node.js scripts
 apply plugin: 'com.moowork.node'
+apply plugin: 'com.wiredforcode.spawn'
+
+//NOTE: The startHsql task can be executed after ant initportal because it uses hsqldb-2.3.2.jar in classpath, otherwise it will fail 
+task startHsql(type: SpawnProcessTask) {
```

Comment:
Instead of "ant initportal" it is more precise to say "uportal-war is built" because it can be built with a few ant commands including initportal and directly with maven.